yaml/wp_base.yaml

```diff
@@ -240,6 +240,15 @@ time:
           - sensor.template.publish:
               id: EL_AUFNAHMELEISTUNG_HEIZ_SUMME_KWH
               state: 0.0
+          - sensor.template.publish:
+              id: WAERMEERTRAG_WW_SUMME_TAG_KWH
```

Comment:
I think we will run into the same problem as with the EL_AUFNAHMELEISTUNG ... the heat pump counts Wh until they overflow at 1000 Wh, then increase kWh. kWh_TAG is reset every day at midnight, but Wh is not :neutral_face: 
![image](https://github.com/user-attachments/assets/b80d00e3-ecad-48fd-ae36-3652ad642363)
![image](https://github.com/user-attachments/assets/ee616c12-998d-4279-8293-0f7c13d373f6)

I fixed it somewhat for the EL_AUFNAHMELEISTUNG but there are still problems ... 
https://github.com/kr0ner/OneESP32ToRuleThemAll/issues/26
I will follow up on this. Maybe we can fix it once for all readings ? :shrug:

Reply:
Frankly speaking I did not understand why you implemented this correction thing. I have no problem to implement it I just don't know why you do it the first place. 

To be on the save side I will replicate your OFFSET_EL_AUFNAHMELEISTUNG_WW_TAG_WH code and commit it again.

Reply:
So for you the Wh is reset at midnight automatically? Cause for me this is dragged over to the next day, which is wrong

Reply:
No it does not. I just did not notice it yet.

Reply:
Good 😅 I'll try to implement a fix for it tonight

Reply:
@Hotdogxxxx this should be a god base for your changes https://github.com/kr0ner/OneESP32ToRuleThemAll/pull/28